Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -319,6 +320,42 @@ theorem apply_one (x : R) : (1 : MulRingNorm R) x = if x = 0 then 0 else 1 :=
 instance : Inhabited (MulRingNorm R) :=
   ⟨1⟩
 
+
+/-- Two multiplicative ring norms `f, g` on `R` are equivalent if there exists a positive constant
+  `c` such that for all `x ∈ R`, `(f x)^c = g x`.
+  This could be generalised to RingNorm, but MulRingNorm does not extend this. -/
+def equiv {R : Type*} [Ring R] (f : MulRingNorm R) (g : MulRingNorm R) :=
```

Comment:
You can add a 
```lean
variable {R : Type*} [Ring R]
```
line before this declaration and avoid having to repeat it everywhere.

Reply:
Added as suggested